maven/build_test.go

```diff
@@ -169,6 +169,33 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 		Expect(result.BOM.Entries).To(HaveLen(0))
 	})
 
+	it("contributes mvnd distribution for API 0.7+", func() {
```

Comment:
If you have a test that sets/unsets env variables, it should have its own context and do that in a Before/After block so that they are guaranteed to be set/unset.

Ex:

```
context("BP_FOO is set", func() {
		it.Before(func() {
			Expect(os.Setenv("BP_FOO", "bar")).To(Succeed())
		})

		it.After(func() {
			Expect(os.Unsetenv(("BP_FOO"))).To(Succeed())
		})
                
                it("contributes mvnd distribution for API 0.7+", func() { .. }
}
...
```